test/scan.test.ts

```diff
@@ -57,6 +57,15 @@ describe('scan', () => {
             remoteUrl: expect.any(String),
             branch: expect.any(String),
           },
+          analytics: [
+            {
+              data: {
+                totalFileSignatures: 3,
+                totalSecondsElapsedToGenerateFileSignatures: 0,
```

Comment:
This could potentially lead to flaky tests. Since we are not aiming to test the actual time value here, I would suggest using `expect.any(Number)` instead.

Reply:
Good point 👍

Reply:
I am keeping totalFileSignatures number as it might not change unless we change the fixtures, regarding the totalSecondsElapsedToGenerateFileSignatures, I've replaced it with `expect.any(Number)`